web/styles/popovers.css

```diff
@@ -1339,3 +1339,16 @@ ul.navbar-dropdown-menu-outer-list {
         display: block;
     }
 }
+
+.navbar-dropdown-hotkey-hint {
+    color: var(--color-hotkey-hint);
+    text-align: center;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 14px;
+    padding: 2px 5px 3px;
+    border-radius: 3px;
+    border: 1px solid var(--color-hotkey-hint);
+    margin-left: auto;
+}
```

Comment:
49f7f02a0a5fddec6ae95eb029a31429bba4fcd5 appears to have added similar CSS to `zulip.css` for this; probably we should just delete that in favor of your version in this PR?
```                                                                                                        
.keyboard-shortcut-option {                                                                             
    & span.tooltip-hotkey-hint {                                                                        
        margin-left: auto;                                                                              
        padding: 0 4px;                                                                                 
        color: var(--color-hotkey-hint);                                                                
        border-color: var(--color-hotkey-hint);                                                         
    }                                                                                                   
}                                                                                                       
```

Reply:
Yeah, done.